pkg/networkservice/common/mechanisms/memif/client.go

```diff
@@ -0,0 +1,95 @@
+// Copyright (c) 2020 Doc.ai and/or its affiliates.
```

Comment:
I'm a bit confused why this is here... what is it attaching to the memif?  You need something (vpp or something else) to attach to memif...

Reply:
OK, guess it is not the right place/time for the memif client :)